game/core/src/me/lihq/game/GameMain.java

```diff
@@ -164,18 +168,94 @@ public void initialiseAllPeople()
         player = new Player("Test name", "player.png", 3, 6);
         player.setRoom(gameMap.getRoom(0));
 
+        //TODO: Add NPC assets
+        NPC npc = new NPC("Colin", "colin.png", 15, 17, gameMap.getRoom(0), true);
+        NPCs.add(npc);
+
+        NPC npc2 = new NPC("Diana", "diana.png",4,4, gameMap.getRoom(1), true);
+        NPCs.add(npc2);
+
+        NPC npc3 = new NPC("Lily", "lily.png", 0, 0, gameMap.getRoom(0), true);
+        NPCs.add(npc3);
+
+        NPC npc4 = new NPC("Mary", "mary.png", 0, 0, gameMap.getRoom(0), true);
+        NPCs.add(npc4);
+
+        NPC npc5 = new NPC("Mike", "mike.png", 0, 0, gameMap.getRoom(0), true);
+        NPCs.add(npc5);
+
+        NPC npc6 = new NPC("Will", "will.png", 0, 0, gameMap.getRoom(0), true);
+        NPCs.add(npc6);
+
+        int amountOfRooms = gameMap.getAmountOfRooms();
+
+        List<Integer> roomsLeft = new ArrayList<Integer>();
+
+        for (int i = 0; i < amountOfRooms; i ++)
         {
-            //TODO: Add NPC assets
-            NPC npc = new NPC("Bill", "player.png", 4, 4, 1, true);
+            roomsLeft.add(i);
+        }
+
+        for (NPC loopNpc : NPCs)
+        {
+            /*
+            Refill the rooms left list if there are more NPCs than Rooms. This will put AT LEAST one NPC per room if so.
```

Comment:
does this dulplicate NPC's essentially? if this is the case I would take this code out as at this stage because we only have to implement 6 npc's I think we can have some empty rooms